site/index.js

```diff
@@ -12,26 +12,59 @@ if (navigator.appVersion.indexOf('Win') !== -1) {
 
 const osLabel = { windows: 'Windows', osx: 'Mac', linux: 'Linux' }[os]
 
+const dlGeneral = document.getElementById('dl-general')
+const dlLinux = document.getElementById('dl-linux')
+const dlWin = document.getElementById('dl-win')
+const dlOSX = document.getElementById('dl-osx')
+
 fetch(REL_URL)
   .then(res => res.json())
   .then(res => {
     const lastVersion = res[0]
     document.getElementById('dl-btn-label').innerHTML = `&nbsp;- ${lastVersion.tag_name}`
     document.getElementById('dl-dl-dl').innerHTML = `Download for ${osLabel}`
+    
+    const { name, assets } = lastVersion
+
+    const linuxName = `mjml-app-${name}-linux-x64.tar.gz`
+    const winName = `mjml-app-win-x64_${name}.exe`
+    const osxName = `mjml-app-${name}-mac.dmg`
+    
+    let linuxUrl, winUrl, osxUrl
+    
```

Comment:
Not using Array.find in case of ie
I could have just concatenated the name to base url but it should be safer this way